test/test-network/systemd-networkd-tests.py

```diff
@@ -6328,6 +6329,21 @@ def test_ipv6_route_prefix(self):
         output = check_output(*networkctl_cmd, '--json=short', 'status', env=env)
         check_json(output)
 
+        output = check_output(*networkctl_cmd, '--json=short', 'status', 'veth-peer', env=env)
+        check_json(output)
+
+        # PREF64 or NAT64
+        j = json.loads(output)
```

Comment:
Could you try something like
```python
pref64 = json.loads(s)['NDisc']['PREF64'][0]

prefix = socket.inet_ntop(socket.AF_INET6, bytearray(pref64['Prefix']))
self.assertEqual(prefix, '64:ff9b::')

prefix_length = pref64['PrefixLength']
self.assertEqual(prefix_length, 96)
```
?

Reply:
This fails in my machine
```
❯ ./b.py
Traceback (most recent call last):
  File "/home/sus/tt/pref65-test/./b.py", line 12, in <module>
    pref64 = json.loads(x)['NDisc']['PREF64'][0]
             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
KeyError: 0

```

```
#!/usr/bin/env python3

import json
import subprocess
import socket

# some JSON:
x = subprocess.check_output("build/networkctl status veth99 --json=short", shell=True);

pref64 = json.loads(x)['NDisc']['PREF64'][0]

prefix = socket.inet_ntop(socket.AF_INET6, bytearray(pref64['Prefix']))
print(prefix)


prefix_length = pref64['PrefixLength']
print(prefix_length)


```

Reply:
My guess would be that b677e5cc5fdd7f2dc360088659138b0a643e71df isn't included in your local build. Could you rebuild networkd with that commit included and try again?